index.d.ts

```diff
@@ -1,20 +1,7 @@
-import nock = require('nock')
 import tape = require('tape')
+import tapeNock = require('./modified_tape')
 
 export = tapeNockFactory
 
-declare function tapeNock (name: string, cb: tape.TestCase): void
-declare function tapeNock (name: string, opts: tape.TestOptions, cb: tape.TestCase): void
-declare function tapeNock (cb: tape.TestCase): void
-declare function tapeNock (opts: tape.TestOptions, cb: tape.TestCase): void
-declare namespace tapeNock {
-  export let nock: typeof nock & {
-    cleanAll: typeof nock.cleanAll
-    disableNetConnect: typeof nock.disableNetConnect
-    enableNetConnect: typeof nock.enableNetConnect
-    recorder: nock.Recorder
-  }
-}
-
 declare function tapeNockFactory (tapeTest: typeof tape, nockOpts: { fixtures: string, mode?: string }): typeof tapeNock
```

Comment:
If we remove nock property from tape, we can just return `typeof tape` and if nock will be exported as `exports.nock = nock` typing will be:

```typescript
import nock2 = require('nock')
declare function tapeNockFactory(...): typeof tape
declare namespace tapeNockFactory {
  export let nock = nock2
}```
And remove `modified_tape.d.ts`